.github/workflows/cts_ci.yml

```
@@ -24,10 +25,27 @@ jobs:
           submodules: true
       - name: Configure CMake
         working-directory: ${{ env.container-workspace }}
-        run: bash /scripts/configure.sh
+        run: |
+          bash /scripts/configure.sh \
+            -DSYCL_CTS_EXCLUDE_TEST_CATEGORIES="${{ env.container-workspace }}/ci/${{ matrix.sycl-impl }}.filter" \
+            -DSYCL_CTS_MEASURE_BUILD_TIMES=ON
       - name: Build 'oclmath'
         working-directory: ${{ env.container-workspace }}/build
         run: cmake --build . --target oclmath
       - name: Build 'util'
         working-directory: ${{ env.container-workspace }}/build
         run: cmake --build . --target util
+      - name: Build all supported test categories
+        working-directory: ${{ env.container-workspace }}/build
+        run: |
+          TS_BEFORE=$(date +%s)
+          cmake --build . --target test_all -- -j${{ env.parallel-build-jobs }}
+          TS_AFTER=$(date +%s)
+          ELAPSED=$(($TS_AFTER - $TS_BEFORE))
+          sort -rn -o build_times.log build_times.log
```

In general avoid abbreviations now we do no longer rely on punch tapes to exchange our open-source programs (which I did long time ago... :-) )
```suggestion
          sort --numeric-sort --reverse --output=build_times.log build_times.log
```

---

Yes you are right, I've addressed this as part of #197, which is where this change is coming from!